src/common/utxobased/engine/UtxoEngineState.ts

```diff
@@ -269,36 +269,36 @@ export function makeUtxoEngineState(
           format,
           changeIndex: branch
         }
-        const branchAddressCount = processor.numAddressesByFormatPath(
+        const branchAddressCount = dataLayer.numAddressesByFormatPath(
           changePath
         )
-        // If the processor has not processed any addresses then the loop
+        // If the DataLayer has not processed any addresses then the loop
         // condition will only iterate once when branchAddressCount is 0 for the
         // first address in the derivation path.
         for (
           let addressIndex = 0;
           addressIndex < branchAddressCount;
           addressIndex++
         ) {
-          const processorAddress = await processor.fetchAddress({
+          const addressData = await dataLayer.fetchAddress({
             format,
             changeIndex: branch,
             addressIndex
           })
-          if (processorAddress == null) {
+          if (addressData == null) {
             throw new Error(
-              `Missing processor address ${format} ${branch} ${addressIndex} during initialization`
+              `Missing data-layer address data with '${format}/${branch}/${addressIndex}' address path during initialization`
```

Comment:
"Missing data-layer address with ${...} path during initialization". There is no need to duplicate so many data address address data words.